src/libraries/System.Net.Http/src/System/Net/Http/WasiHttpHandler/WasiHttpHandler.cs

```diff
@@ -471,7 +471,6 @@ private static Task RegisterWasiPollable(IPoll.Pollable pollable)
             GC.SuppressFinalize(pollable);
 
             return CallRegisterWasiPollableHandle((Thread)null!, handle);
```

Comment:
```suggestion
            return CallRegisterWasiPollableHandle((Thread)null!, handle);

```
Reverting whitespace changes.